model_analyzer/config/input/config_command_profile.py

```diff
@@ -477,6 +478,24 @@ def _add_run_search_configs(self):
                 description=
                 "Max instance count value that run config search should not go beyond that."
             ))
+        self._add_config(
+            ConfigField(
+                'run_config_search_max_model_batch_size',
+                flags=['--run-config-search-max-model-batch-size'],
+                field_type=ConfigPrimitive(int),
+                default_value=DEFAULT_RUN_CONFIG_MAX_MODEL_BATCH_SIZE,
+                description=
+                "Max value for the model's max_batch_size that run config search should not go beyond."
```

Comment:
Get rid of Max at the start of the description: "Value for the ...."
And change "should" to "will"